.pre-commit-config.yaml

```diff
@@ -20,7 +20,9 @@ repos:
     -   id: check-yaml
     -   id: debug-statements
         exclude: _pytest/debugging.py
+        language_version: python3
     -   id: flake8
+        language_version: python3
```

Comment:
hmmm why was this necessary?

Reply:
it used python2 on my box and crashed the flake8 builds on incompatible syntax

Reply:
we do on a few scripts (like the one to make releases
the thing is - when pre-commit is installed using python3 the issue doesn't seem to happen, when it is installed with python2, it crashed